package/rmservewacominput/rmservewacominput_gui

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+/opt/bin/appmarkable /opt/bin/rmservewacominput -n rmServeWacomInput -i /opt/etc/draft/icons/rmservewacominput.png
```

Comment:
Adding `exec` will replace the bash process with the appmarkable process instead of making it a child of the bash process.

```suggestion
exec /opt/bin/appmarkable /opt/bin/rmservewacominput -n rmServeWacomInput -i /opt/etc/draft/icons/rmservewacominput.png
```

Reply:
Good point! Wasn't sure either if this would pose a problem.